instances/widgets.treasury-factory.near/widget/pages/settings/Thresholds.jsx

```diff
@@ -124,6 +162,12 @@ useEffect(() => {
   }
 }, [isTxnCreated, lastProposalId]);
 
+useEffect(() => {
+  if (lastProposalId) {
+    fetchProposals();
+  }
+}, [lastProposalId]);
+
```

Comment:
just to make sure it doesn't fetch too many times
```suggestion
useEffect(() => {
  if (lastProposalId && !proposals.length) {
    fetchProposals();
  }
}, [lastProposalId]);

```